### DISCUSSION TITLE
Why is the FlowPart not reusable?

### DISCUSSION THREAD

**Discussion by devlife** [OrchardCMS/OrchardCore#15424] [Q&A]
Is there a reason the FlowPart is not reusable?

**Comment by hishamco** [comment]
Could you please elaborate? What's your case?

**Comment by devlife** [comment]
We use FlowParts on most pages including where there are other fields/parts on the page. Sometimes we want to do something like this:

```
Top FlowPart

Field1   Field2
Field 3

Bottom FlowPart

...other fields.
```

However, it appears that we can't add named FlowParts. I was really just wondering why.

**Reply by hishamco** [reply]
@ns8482e is this by design?

**Reply by ns8482e** [reply]
I guess FlowPart can't be used as named part.

**Comment by Skrypt** [comment]
You would need to create reusable FlowPart templates. And then select them from these pages. That's not something we currently have in OC.
